<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginStudentTable</class>
 <widget class="QDialog" name="CLoginStudentTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>当前登录学员</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="viewLoginStudent"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>100</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonTask">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>设置任务</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonScene">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>设置工况</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFault">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>设置故障</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRefresh">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExit">
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonTask</tabstop>
  <tabstop>buttonExit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>CLoginStudentTable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
